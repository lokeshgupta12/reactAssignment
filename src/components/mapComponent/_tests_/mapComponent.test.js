import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import MapWithADirectionsRenderer from '../index';
import Adapter from 'enzyme-adapter-react-16';

// Configure Enzyme adapter for Jest
Enzyme.configure({ adapter: new Adapter() });

describe('<MapWithADirectionsRenderer /> component', () => {

  it('should match the snapshot', () => {
    const tree = renderer.create(<MapWithADirectionsRenderer direction={true}/>).toJSON();

    expect(tree).toMatchSnapshot();
  });

})