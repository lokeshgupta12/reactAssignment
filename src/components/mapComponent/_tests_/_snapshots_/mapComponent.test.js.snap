// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapWithADirectionsRenderer /> component should match the snapshot 1`] = `
  <div
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;